src/init.cpp

```diff
@@ -489,6 +489,8 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-bytespersigop", strprintf(_("Minimum bytes per sigop in transactions we relay and mine (default: %u)"), DEFAULT_BYTES_PER_SIGOP));
     strUsage += HelpMessageOpt("-datacarrier", strprintf(_("Relay and mine data carrier transactions (default: %u)"), DEFAULT_ACCEPT_DATACARRIER));
     strUsage += HelpMessageOpt("-datacarriersize", strprintf(_("Maximum size of data in data carrier transactions we relay and mine (default: %u)"), MAX_OP_RETURN_RELAY));
+    // called excessiveblocksize in BU
+    strUsage += HelpMessageOpt("-blocksizeacceptlimit=<n>", strprintf("This node will not accept blocks of this size or larger. Unit is multiples of 100KB (default: %u)", DEFAULT_BLOCK_ACCEPT_SIZE));
```

Comment:
Looking at the code, blocks of this size are accepted, but not larger, which is what you'd expect from a "limit".
For the doc string?